rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/ip/DiscoverHardcodedIPAddressRuleProvider.java

```diff
@@ -42,7 +42,7 @@
 @RuleMetadata(phase = MigrationRulesPhase.class, tags = {"cloud-readiness"})
 public class DiscoverHardcodedIPAddressRuleProvider extends AbstractRuleProvider
 {
-    private static final String IP_PATTERN = "(?<![\\w.])\\d{1,3}\\.\\d{1,3}\\.\\d{1,3}\\.\\d{1,3}(?![\\w.])";
+    private static final String IP_PATTERN = "(?<![\\w./-])\\d{1,3}\\.\\d{1,3}\\.\\d{1,3}\\.\\d{1,3}(?![\\w.-])";
```

Comment:
So basically it stops the match if it is preceded by "-" or followed by "-"?

Reply:
And also the `/` as in `Browser/5.0.4.1`.
I haven't found any other case of a false positive in the more than 800 analyzed issues, other than `server.number=8.0.30.0`

Reply:
Oh, I missed that the "/" was new too. Ok, this looks good to me.